.gitignore

```diff
@@ -1,3 +1,6 @@
 node_modules
 design
-.DS_Store
\ No newline at end of file
+.DS_Store
+.env*
+!.env.sample
```

Comment:
I moved .env to .env.sample as a general good practice preventing secrets to be committed. This change ignores every file that start with .env except .env.sample.